app/views/static_pages/room.html.erb

```diff
@@ -57,25 +55,26 @@ var comm = new Icecomm('9gv3ODZorBS2kVuNwnugBeV432PcNX3Qn0HmW33ofxK/u5Xolu');
 
     comm.on('disconnect', function(options) {
       document.getElementById(options.callerID).remove();
-
+      comm.close();
     });
+
     comm.on('data', function(options) {
     console.log("data: ",options.data);
-    //appendNewIM(options);
+    appendNewIM(options);
   });
 
-   comm.on('data',appendNewIM);
```

Comment:
replaced line 67 - just to clean up the code, didn't need to respond to data event twice.